### PR TITLE
Multiple improvements to SLES4SAP tests

### PR DESCRIPTION
**Pull request by alvarocarvajald**
This PR features the following improvements to the sles4sap tests:

- Removal of ^ and $ in saptune/sapconf regular expressions so that it doesn't produce false positives on tests runs when the serial console gets contaminated with extra characters as in https://openqa.suse.de/tests/1405642#step/saptune/17 or 
https://openqa.suse.de/tests/1404143#step/sapconf/13
- Adds SAP Profile name verification after profile changes in sapconf test.
- Sets NetWeaver profile and solution before NetWeaver's ASCS installation.
- Adds support for multiple ARCHs in NetWeaver's ASCS installation with a single shared volume passed on the NW variable so there's no need for architecture specific test suites.
- Changes value of SAPINST_SLP_MODE to false (required by NetWeaver's 7.42 unattended installation as documented in https://launchpad.support.sap.com/#/notes/0002575377).

- Related ticket: N/A
- Needles: N/A
- Verification run: http://mango.suse.de/tests/195

### CODE REVIEW

File: tests/sles4sap/nw_ascs_install.pm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # SUSE's openQA tests
 #
-# Copyright © 2017 SUSE LLC
+# Copyright © 2018 SUSE LLC
```
Comment (asmorodskyi) [inline tests/sles4sap/nw_ascs_install.pm]: better 2017-2018

File: tests/sles4sap/sapconf.pm [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # SUSE's openQA tests
 #
-# Copyright © 2017 SUSE LLC
+# Copyright © 2018 SUSE LLC
```
Comment (asmorodskyi) [inline tests/sles4sap/sapconf.pm]: 2017-2018

File: tests/sles4sap/nw_ascs_install.pm (hunk 1 above)
Comment (alvarocarvajald) [reply tests/sles4sap/nw_ascs_install.pm]: Done.

File: tests/sles4sap/sapconf.pm (hunk 2 above)
Comment (alvarocarvajald) [reply tests/sles4sap/sapconf.pm]: And done.

**Comment by asmorodskyi** [comment]
LGTM , but waiting for green light from Travis

**Comment by ldevulder** [comment]
Looks good.

**Pull request merged**